bootstrap/pkg/apis/apps/configconverters/converters.go

```diff
@@ -145,7 +145,20 @@ func isCwdEmpty() string {
 	return cwd
 }
 
-func WriteConfigToFile(config kfconfig.KfConfig, filename string) error {
+func WriteConfigToFile(config kfconfig.KfConfig) error {
```

Comment:
Do you want to take a pointer so that you can potentially change ConfigFileName?
My expectation would be that LoadFromURI sets ConfigFileName originally to the URI e.g.
https://.../kfctl_gcp_iap.yaml

Then somewhere in the code base we would replace that with the local file path.

A natural place to do that would be in WriteConfigToFile.

Reply:
doing it in LoadConfigFromURI